float32 abs_position
float32 velocity
---
float32 current_position
bool reached_position
---
float32 current_position
float32 current_velocity
float32 motor_current